cpp/src/dbscan/runner.h

```diff
@@ -37,9 +37,9 @@ static const int TPB = 256;
  * 1. Turn any labels matching MAX_LABEL into -1
  * 2. Subtract 1 from all other labels.
  */
-template <typename Type>
-__global__ void relabelForSkl(Type* labels, Type N, Type MAX_LABEL) {
-    int tid = threadIdx.x + blockDim.x * blockIdx.x;
+template <typename Type, typename Index_ = int>
+__global__ void relabelForSkl(Type* labels, Index_ N, Type MAX_LABEL) {
+    Index_ tid = threadIdx.x + blockDim.x * blockIdx.x;
```

Comment:
No, this was done to keep the warnings away that happens at line 44 below. Besides, the template params were improper before. ie., `N` should not be of the type `Type`, but of type `Index_`, since it is used for indexing purposes. What say?